src/main/java/duke/Event.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Encapsulates a {@link Task} that will be done in a fixed time slot.
+ */
+public class Event extends Task {
+    private final LocalDate time;
+
+    /**
+     * Creates an Event
+     *
+     * @param content description of the event
```

Comment:
I think the formatting could be improved if the Javadocs started with capital letters in sentences.